fmriprep/interfaces/freesurfer.py

```diff
@@ -222,6 +264,14 @@ def _list_outputs(self):
         return outputs
 
 
+class PatchedBBRegisterRPT(TruncateLTA, BBRegisterRPT):
+    lta_outputs = ['out_lta_file']
```

Comment:
We can just use `pass` here, since it's the default `lta_outputs` from `TruncateLTA`.